### PR TITLE
Editorial changes and fixes for Thomas Fossati's ARTART review

### PR DESCRIPTION
**Pull request by cabo**
Thank you!

### CODE REVIEW

**Review by thomas-fossati — APPROVED** [review]
👍

File: draft-ietf-cbor-time-tag-extracted.cddl [hunk 1]
```diff
@@ -94,20 +94,20 @@ Duration = #6.1002(etime-detailed)
...
-  (start: ~Time,
-   ((end: ~Time,
+  (start: ~Etime,
+   ((end: ~Etime,
      ? duration: null) //
     (end: null,
      duration: ~Duration))) //
   (start: null,
-   end: ~Time,
+   end: ~Etime,
```
Comment (henkbirkholz) [inline draft-ietf-cbor-time-tag-extracted.cddl:110]: :+1:

File: draft-ietf-cbor-time-tag.md [hunk 2]
```diff
@@ -208,10 +208,12 @@ Terminology         {#terms}
...
-Where bit arithmetic is explained, this document uses the notation
-familiar from the programming language C (including C++14's 0bnnn
-binary literals), except that the operator "\*\*" stands for
-exponentiation.
+
+Superscript notation denotes exponentiation.  For example, 2 to the
+power of 64 is notated: 2<sup>64</sup>.  In the plain-text rendition
+of this specification, superscript notation is not available and
+exponentiation therefore is rendered by the surrogate notation seen
+here in the plain-text rendition.
```
Comment (henkbirkholz) [inline draft-ietf-cbor-time-tag.md:216]: :+1:

File: draft-ietf-cbor-time-tag.md [hunk 3]
```diff
@@ -329,14 +332,15 @@ etime-detailed = ({
   * (uint .feature "etime-critical-extension") => any
 }) .within etime-framework
 ~~~
-
+{: #tag-1001 title="CDDL definition of Tag 1001"}
```
Comment (henkbirkholz) [inline draft-ietf-cbor-time-tag.md:335]: better

File: draft-ietf-cbor-time-tag.md [hunk 4]
```diff
@@ -309,7 +312,7 @@ This document does not define supplementary text keys.
 A number of both unsigned and negative-integer keys are defined in
 the following subsections.
 
-A formal definition of Tag 1001 in CDDL is:
+{{tag-1001}} provides a formal definition of Tag 1001 in CDDL.
```
Comment (henkbirkholz) [inline draft-ietf-cbor-time-tag.md:315]: yes, see 335 comment

File: draft-ietf-cbor-time-tag.md [hunk 5]
```diff
@@ -247,18 +249,19 @@ Intents might include information about time zones, daylight savings
...
-* Direct representation of natural platform time formats.  Some
-  platforms use epoch-based time formats that require some computation
-  to convert them into the representations allowed by tag 1; these
-  computations can also lose precision and cause ambiguities.
-  (The present specification does not take a position on whether tag 1
-  can be "fixed" to include, e.g., Decimal or BigFloat representations.
-  It does define how to use these representations with the extended
-  time format.)
+additional map keys for the map that is the content of the tag (see
+`etime-detailed` in {{tag-1001}}),
```
Comment (henkbirkholz) [inline draft-ietf-cbor-time-tag.md:253]: yes, see 335 comment

File: draft-ietf-cbor-time-tag.md [hunk 6]
```diff
@@ -247,18 +249,19 @@ Intents might include information about time zones, daylight savings
...
+the specification for
+which is referenced by the registry entry (see {{time-format}}).
+
+For example, map keys could be registered for direct representations
+of natural platform time formats.  Some platforms use epoch-based
+time formats that require some computation to convert them into the
```
Comment (henkbirkholz) [inline draft-ietf-cbor-time-tag.md:259]: Non-blocking nit:
```suggestion
of natural platform time formats.  Some platforms use epoch-based
time formats that require computation to convert them into the
```

File: draft-ietf-cbor-time-tag.md [hunk 7]
```diff
@@ -329,14 +332,15 @@ etime-detailed = ({
...
 
 Key 1
 -----
 
-Key 1 indicates a Base Time value that is exactly like the data item that would
-be tagged by CBOR tag 1 (Posix time {{TIME_T}} as int or float).
-The time value indicated by the value under this key can be further
+Key 1 indicates a base time value that is exactly like the data item that would
+be tagged by CBOR tag 1 (POSIX time {{TIME_T}} as int or float).
+As described above, the time value indicated by the value under this
```
Comment (henkbirkholz) [inline draft-ietf-cbor-time-tag.md:342]: :+1:

File: draft-ietf-cbor-time-tag.md [hunk 8]
```diff
@@ -347,10 +351,11 @@ $$ETIME-BASETIME //= (1: ~time)
...
 ------------
 
-Keys 4 and 5 indicate a Base Time value and are like key 1, except that the data item is an array as
-defined for CBOR tag 4 or 5, respectively.  This can be used to include
-a Decimal or Bigfloat epoch-based float {{TIME_T}} in an extended time.
-
+Keys 4 and 5 indicate a base time value and are like key 1, except that the data item is an array as
+defined for CBOR tag 4 or 5, respectively.
+This can be used to include a Decimal or Bigfloat epoch-based float
+{{TIME_T}} in an extended time, e.g., to achieve higher resolution or to
```
Comment (henkbirkholz) [inline draft-ietf-cbor-time-tag.md:357]: :+1:

File: draft-ietf-cbor-time-tag.md [hunk 9]
```diff
@@ -447,7 +454,7 @@ used to determine the point in time.
 
 The first three are analogous to `clock-quality-grouping` in
 {{RFC8575}}, which is in turn based on the definitions in
-{{IEEE1588-2008}}; two more are specific to this document.
+{{IEEE1588-2008}}; the last two are specific to this document.
```
Comment (henkbirkholz) [inline draft-ietf-cbor-time-tag.md:457]: :+1:

**Review by henkbirkholz — APPROVED** [review]

File: draft-ietf-cbor-time-tag.md (hunk 6 above)
Comment (cabo) [reply draft-ietf-cbor-time-tag.md:259]: I actually like the qualification as to what amount of computation is needed (i.e., not much).

File: draft-ietf-cbor-time-tag.md (hunk 4 above)
Comment (cabo) [reply draft-ietf-cbor-time-tag.md:315]: What is 335 comment?

**Pull request merged**